tox.ini

```diff
@@ -13,8 +13,8 @@ deps =
 commands =
   yamllint etc/kayobe
   reno lint
-  doc8 README.rst doc/source --ignore D001
-
+  # secret-rotation must be skipped because it includes purposeful whitespace
+  doc8 README.rst doc/source --ignore D001 --ignore-path doc/source/operations/secret-rotation.rst 
```

Comment:
I think there's an argument to be made for a separate config file here but I think while there's only a single path to skip I prefer adding it here.

Reply:
I didn't, now done